backend/onconnect/src/connect.ts

```diff
@@ -0,0 +1,23 @@
+import * as AWS from 'aws-sdk';
+import { EXPIRY_TIME_IN_HOUR, TABLE_NAME } from './const';
+
+export const connect = async (ddb: AWS.DynamoDB.DocumentClient, connectionId: any) => {
```

Comment:
I assume connectionId does have a data type, even if it is untyped in the event. Would be nice to have the proper type here, even if it means using a type cast in the wrapper.